pkg/kubelet/cm/container_manager.go

```diff
@@ -212,6 +224,32 @@ func int64Slice(in []int) []int64 {
 	return out
 }
 
+func podHasExclusiveCPUs(cr cpuAllocationReader, pod *v1.Pod) bool {
+	for _, container := range pod.Spec.InitContainers {
+		exclusiveCPUs := cr.GetExclusiveCPUs(string(pod.UID), container.Name)
```

Comment:
I think `pod.Spec.InitContainers` has the same type as `pod.Spec.Containers` so you should be able to reuse `containerHasExclusiveCPUs` here as well.

Reply:
resolved!